src/lib/Scenes/Search/Search.tsx

```diff
@@ -39,7 +40,7 @@ export const Search: React.FC = () => {
             </Flex>
           </TouchableWithoutFeedback>
         </Flex>
-        <AutosuggestResults query={query} />
+        {query.trim() ? <AutosuggestResults query={query} /> : <RecentSearches />}
```

Comment:
Do we want to move the `trim()` outside this context, so that the trim'd version is also included in the `query` prop here?